frontend/views/pages/Join.vue

```diff
@@ -176,7 +173,14 @@ export default ({
         console.error('Join.vue accept() error:', e)
         this.ephemeral.errorMsg = e.message
         this.pageStatus = 'INVALID'
+      } finally {
+        sbp('okTurtles.events/emit', JOIN_GROUP)
       }
+    },
+    acceptPostEffect () {
+      return new Promise(resolve => {
+        sbp('okTurtles.events/once', JOIN_GROUP, resolve)
+      })
```

Comment:
`accpetPostEffect` is for waiting until the user finishes joining group.
If we use `accept` function in the `postSubmit`, the `accept` function would be called twice.

As we can see in `initialize` function, `accept` function is called after the user finishes logging in.
Users could log in by clicking `Sign Up` or `Login` button. So if we call `accept` function in `postSubmit` of those two forms, it will be called twice; once inside `initialize`, once in `postSubmit`.